src/scanner.c

```diff
@@ -359,13 +359,11 @@ openvas_scanner_realloc ()
 /**
  * @brief Write as much as possible from the to_scanner buffer to the scanner.
  *
- * @param[in]  nvt_cache_mode  NVT cache mode.
- *
  * @return 0 wrote everything, -1 error, -2 wrote as much as scanner accepted,
  *         -3 did an initialisation step.
  */
 int
-openvas_scanner_write (int nvt_cache_mode)
+openvas_scanner_write (void)
```

Comment:
```suggestion
openvas_scanner_write ()
```

We use () everywhere else.

Reply:
Reverted accordingly.